tests/PHPStan/Rules/Exceptions/CaughtExceptionExistenceRuleTest.php

```diff
@@ -0,0 +1,29 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Exceptions;
+
+class CaughtExceptionExistenceRuleTest extends \PHPStan\Rules\AbstractRuleTest
```

Comment:
@bogdan-rizac I disagree with the `@deprecated` annotation. PHPStan is still in the pre-1.0 era so each 0.x release can contain BC breaks. Also, there's no reason to extend rule classes.